Sources/TokamakStaticHTML/Views/HTML.swift

```diff
@@ -17,10 +17,35 @@
 
 import TokamakCore
 
+/** Represents an attribute of an HTML tag. To consume updates from updated attributes, the DOM
+ renderer needs to know whether the attribute should be assigned via a DOM element property or the
+ [`setAttribute`](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute) function.
+ The `isUpdatedAsProperty` flag is used to disambiguate between these two cases.
+ */
+public struct HTMLAttribute: Hashable {
+  public let value: String
+  public let isUpdatedAsProperty: Bool
+
+  public init(_ value: String, isUpdatedAsProperty: Bool) {
+    self.value = value
+    self.isUpdatedAsProperty = isUpdatedAsProperty
+  }
+}
```

Comment:
Neat idea! How do you feel about adding a helper constructor so you can just do `.property("some-name")`?

```suggestion

    static func property(_ value: String) -> HTMLAttribute {
    .init(value, isUpdatedAsProperty: true)
  }
}
```